src/components/search/hooks.js

```diff
@@ -0,0 +1,9 @@
+import { useRef, useEffect } from 'react';
+
+export const useIsFirstRender = () => {
```

Comment:
This seems like a potentially useful hook outside of search as well. Is there a more generic, "global" place to put this?

Reply:
True fact! I thought about putting it in `app`, but that didn't seem like the right place for it, or starting a top level `utils` folder, but I didn't feel like that was brilliant either. Do you have feelings about where to put it?

Reply:
Moved to utils, lmk if you think of a better idea. :)

Reply:
Utils is fine for now :)